cms/djangoapps/contentstore/views/item.py

```diff
@@ -128,6 +132,9 @@ def xblock_handler(request, usage_key_string):
               these fields:
                 :parent_locator: parent for new xblock, required for both duplicate and create new instance
                 :duplicate_source_locator: if present, use this as the source for creating a duplicate copy
+                :move_source_locator: if present, use this as the source item for moving
+                :move_dest_locator: if present, use this as the destination parent for moving an item
+                :source_index: if present, use this as the source index for moving an item to a particular index
```

Comment:
It would be good to mention what the default is if this value isn't provided (does it add it at the end?).